generators/server/index.js

```diff
@@ -598,7 +598,7 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
             'java:war': './mvnw -ntp verify -DskipTests --batch-mode -Pwar',
             'java:docker': './mvnw -ntp verify -DskipTests -Pprod jib:dockerBuild',
             'java:docker:arm64': 'npm run java:docker -- -Djib-maven-plugin.architecture=arm64',
-            'backend:unit:test': `./mvnw -ntp${excludeWebapp} verify --batch-mode ${javaCommonLog} ${javaTestLog}`,
+            'backend:test': `./mvnw -ntp${excludeWebapp} verify --batch-mode ${javaCommonLog} ${javaTestLog}`,
```

Comment:
npm scripts should be stable to don’t break existing scripts.
A compatibility script should be added and dropped for v8.

Reply:
:'(. It's not only unit but integration tests too!
What about a 7.7.X? It's not breaking the blueprints API.

What's the guideline about Semantic versioning? I definitely don't get it.

To me, and according to semantic versioning spec:
- Major (user API incompatibility)=> Big change on the generated code (i.e. switching from service to modulith), removal of options or modification an option behavior
- Minor (developer API incompatibility) => Breaking blueprint API
- Patch => Everything else

Reply:
> To me, and according to semantic versioning spec:

That's not what node/npm ecosystem uses, see https://docs.npmjs.com/about-semantic-versioning#incrementing-semantic-versions-in-published-packages.

We should break blueprints only at major versions.
And we usually don't release patch versions.

Reply:
Ahah thanks for the pointer, I'm so old school I still followed the one of OSGi: http://docs.osgi.org/whitepaper/semantic-versioning/040-semantic-versions.html (which could have been nice in our context, if we weren't on js)

Still, this one doesn't break blueprints ^^. I'll make the change though